docs/en/latest/plugins/opa.md

```diff
@@ -0,0 +1,275 @@
+---
+title: opa
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Summary
+
+- [**Description**](#description)
+- [**Attributes**](#attributes)
+- [**Data Definition**](#data-definition)
+- [**Example**](#example)
+
+## Description
+
+The `opa` plugin is used to integrate with Open Policy Agent. By using this plugin, users can decouple functions such as authentication and access to services and reduce the complexity of the application system.
+
+## Attributes
+
+| Name | Type | Requirement | Default | Valid | Description |
+| -- | -- | -- | -- | -- | -- |
+| host | string | required |   |   | Open Policy Agent service host (eg. https://localhost:8181) |
+| ssl_verify | boolean | optional | true |   | Whether to verify the certificate |
+| policy | string | required |   |   | OPA policy path (It is a combination of `package` and `decision`. When you need to use advanced features such as custom response, `decision` can be omitted) |
+| timeout | integer | optional | 60000ms | [1, 60000]ms | HTTP call timeout. |
```

Comment:
Hi, some users report that they have no idea what's the meaning of `[a, b]` 😄 only supporting 1 or 60000? or it means >= 1 and <= 60000?

```suggestion
| timeout | integer | optional | 60000ms | [1, 60000]ms | HTTP call timeout. |
```

Reply:
I think this is exactly in line with the expression of the concept of interval in mathematics, which means => 1 and <= 60000.